<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemViewHeaderConfig</class>
 <widget class="QWidget" name="ItemViewHeaderConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="la_main">
     <item>
      <layout class="QVBoxLayout" name="la_buttons">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="b_first">
         <property name="toolTip">
          <string>Переместить в начало</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../share/icons/share_icons.qrc">
           <normaloff>:/share_icons/up-arrow.svg</normaloff>:/share_icons/up-arrow.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="b_left">
         <property name="toolTip">
          <string>Переместить выше</string>
         </property>
         <property name="text">
          <string> Левее</string>
         </property>
         <property name="icon">
          <iconset resource="../../../share/icons/share_icons.qrc">
           <normaloff>:/share_icons/up-arrow-single.svg</normaloff>:/share_icons/up-arrow-single.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="b_right">
         <property name="toolTip">
          <string>Переместить ниже</string>
         </property>
         <property name="text">
          <string> Правее</string>
         </property>
         <property name="icon">
          <iconset resource="../../../share/icons/share_icons.qrc">
           <normaloff>:/share_icons/down-arrow-single.svg</normaloff>:/share_icons/down-arrow-single.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="b_last">
         <property name="toolTip">
          <string>Переместить в конец</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../share/icons/share_icons.qrc">
           <normaloff>:/share_icons/down-arrow.svg</normaloff>:/share_icons/down-arrow.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../share/icons/share_icons.qrc"/>
 </resources>
 <connections/>
</ui>
